server/controllers/activation.controller.js

```diff
@@ -0,0 +1,51 @@
+const User = require('../../config/db').User;
+const Profile = require('../../config/db').Profile;
+const jwt = require('jsonwebtoken');
+const {mailer, message} = require('./../helper');
+const secret = require('./../../config/jwt.secretkey.json');
+const activUser = require(`./../../config/config.json`).activUser;
+
+module.exports = {
+  activation(req, res) {
```

Comment:
which controller will be better for use?...or maybe it doesn't matter...because route still stay same as is exists?